### DISCUSSION TITLE
Hey guys, In which course I learn to start auditing and participate in codehawks

### DISCUSSION THREAD

**Discussion by vasanthmarimuth** [Cyfrin/foundry-full-course-cu#2703] [Q&A] [answered]
I'm interested in participating codehawks, so I need to know when I learn about it in the course.

**Comment by EngrPips** [comment, marked as answer]
This course https://updraft.cyfrin.io/courses/security

**Reply by vasanthmarimuth** [reply]
Thanks @EngrPips

**Reply by EngrPips** [reply]
You are welcome buddy.
